dataset.py

```diff
@@ -417,6 +418,8 @@ def create_fraudulent_samples(real_samples_data: pd.DataFrame, mean_isoscapes: l
   count = 0
   lab_samp = real_samples
 
+  fake_amount = round(simulated_fraud_percent*real_samples_code.ngroups)
```

Comment:
i think this should be:
fraud_count / (fraud_count + real_count ) = simulated_fraud_percent
solving for fraud_count
fraud_count = simulated_fraud_percent * fraud_count + simulated_fraud_percent * real_count
fraud_count - simulated_fraud_percent * fraud_count = simulated_fraud_percent * real_count
fraud_count (1 - simulated_fraud_percent )= simulated_fraud_percent * real_count

fraud_count= simulated_fraud_percent * real_count /  (1 - simulated_fraud_percent )

Reply:
We want to get a fraud sample by groups. Using counts is not reliable for samples who happen to, for some element, to be missing (we end up with 3/4). What